bokeh/plotting/tests/test_helpers.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+
+import pytest
+
+import bokeh.plotting.helpers as ph
```

Comment:
No need to change this, but why not `from bokeh.plotting.helpers import _convert_responsive`?